### PR TITLE
Add search

### PR DESCRIPTION
**Pull request by loopDelicious**
@godfrzero / @abhijitkane please don't merge. can you please take a look at this? having trouble with the front end search display. algolia took a look and the code is working on their jsfiddle: https://jsfiddle.net/bobylito/0okx8ut7/. Do you know why this would NOT work with our config?

### CODE REVIEW

**Comment by loopDelicious** [comment]
@godfrzero could you please review?

Add search to home page and sidebar.

File: _includes/_search.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<script>
+
+'use strict';
+/* global instantsearch */
+
+const search = instantsearch({
```
Comment (godfrzero) [inline _includes/_search.html]: Not sure how stable `const` is in the wild. AFAIK, browser behavior will still be a bit flaky: some will treat it as a var, others will treat it as a constant but within a function scope.

If this ends up causing a problem at some point, it wouldn't hurt to just replace it with `var`

File: _includes/_search.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+);
+
+const container = document.querySelector('#pagination');
+const paginationWidget = instantsearch.widgets.pagination({
+    container: '#pagination',
+    cssClasses: {
+        root: ['pagination', 'search-results'],
+        active: 'active'
+    },
+    hitsPerPage: 5
```
Comment (godfrzero) [inline _includes/_search.html]: Why `5` here and `6` in the previous case?

File: _includes/_search.html (hunk 1 above)
Comment (loopDelicious) [inline _includes/_search.html]: Makes sense, replaced all.

File: _includes/_search.html (hunk 2 above)
Comment (loopDelicious) [inline _includes/_search.html]: `hitsPerPage` doesn't seem to do what I'd expect, will remove property in both instances.

**Pull request merged**